### ISSUE TITLE
[HIGH] Gak ada inovasi — cuma YOLO fine-tune vanilla

### ISSUE THREAD

**Issue by jaweed3** [jaweed3/RescueVision#9]
## Problem
Apa yang kita lakuin literally:
```python
model = YOLO('yolov8n.pt')
model.train(data='dataset.yaml', epochs=100, imgsz=640, batch=16)
```

Ini template bawaan Ultralytics. Di CV track, puluhan tim submit YOLO fine-tune. Yang menang punya **twist**.

Hal yang gak kita lakukan:
- Custom architecture modification (attention, feature fusion)
- Test-Time Augmentation
- Multi-scale inference pipeline
- Ensemble methods (weighted box fusion)
- Knowledge distillation (teacher → student)
- Pruning / quantization agresif
- Pseudo-labeling / self-training
- Domain adaptation techniques
- Custom loss function (e.g., focal loss tuning, small-object weighted loss)

## Root cause
Kita cuma kejar constraint compliance, gak push innovation.

## Task list
- [ ] Research paper terbaru: small object detection in drone imagery (2023-2025)
- [ ] Implement minimal 3 improvement dari paper (contoh: SAHI, TTA, ensemble)
- [ ] Ablation study — mana improvement yang paling impactful
- [ ] Quantization: INT8 vs FP16 vs FP32 latency + accuracy tradeoff
- [ ] Evaluate edge deployment readiness (onnxruntime, OpenVINO, TensorRT comparison)

## Acceptance criteria
- Minimal 2 teknik improvement beyond vanilla YOLO di-implement dan di-dokumentasi
- Ablation table: kontribusi tiap teknik terhadap mAP
